libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -508,6 +532,12 @@ class BreezSDKMapper {
             "htlcMaximumMsat": routeHintHop.htlcMaximumMsat
         ]
     }
+
+    static func dictionaryOf(signMessageResponse: SignMessageResponse) -> [String: Any?] {
```

Comment:
`signMessageResponse.signature` should always return a value so the return type can be `[String: Any]`